docs/services/qix-engine/access-control.md

```diff
@@ -72,36 +83,41 @@ docker run -v <host folder path>:<container folder path> qlikcore/engine:<versio
     -S SystemDenyRulePath=/<container folder path>/deny.txt
 ```
 
-### More Code Examples
+For more examples using ABAC, see the
+[core-authorization](https://github.com/qlik-oss/core-authorization) repository,
+which contains running code examples on how to enable ABAC and how to provide rules to Qlik Associative Engine.
 
-The [core-authorization](https://github.com/qlik-oss/core-authorization) repository contains running code examples on
-how to enable ABAC and how to provide rules to Qlik Associative Engine.
+## Rules language
 
-## Rules Language
+Before you write any ABAC rules,
+it is imporatant to understand the syntax
+and semantics of the rules language,
+and how they interact with each other.
 
-The following sections provide a more in-depth documentation of the rules language in Qlik Associative Engine - its
-syntax and semantics.
+### Rules language concepts
 
 Rules are built upon three concepts:
 
-* **User** - The _subject_ that shall be granted or denied access.
+* **User** - The _subject_ that is granted or denied access.
 * **Resource** - The _object_ to which actions are granted or denied.
-* **Action** - The operation a user wants to perform on a resource.
+* **Action** - The operation performed on a resource.
 
-All of these can be expressed and used in the rules language.
+These concepts are used to build the expressions in the rules language.
 
-### Expressions
+### Rules language expressions
 
-Rules, in which the entities above are used, are written as Boolean expressions, evaluating to `true` or `false`.
-Expressions are written using _logical_ and _comparison_ operators.
+Rules are made of expressions, and these expressions
+are written with logical and comparison operators.
+A rule evaluates to `true` when all of the expressions are true.
+A rule evaluates to `false` when any expression is not true.
```

Comment:
This is not really correct since you can write rules that contain the `or` operator, in which case the rule may evaluate to `true` even though not all sub expressions do.